.github/ISSUE_TEMPLATE/question.yml

```diff
@@ -13,7 +13,7 @@ body:
           required: true
         - label: Searched open and closed [GitHub issues](https://github.com/pingcap/ticdc/issues?q=is%3Aissue)
           required: true
-        - label:  Read the [documentation](https://docs.pingcap.com/tidb/stable/ticdc-overview)
+        - label:  Read the [documentation](https://docs.pingcap.com/tidb/stable)
```

Comment:
not sure why we changed the URL

Reply:
Since this is a shared template, it may not be appropriate to jump directly to ticdc's documentation.